presentations/2023-06-02--eth-belgrade--milos/README.md

```diff
@@ -0,0 +1,165 @@
+# Gnolang workshop: Alice in Gno.land
+
+The recording of this workshop is available [here](#).
+
+## Requirements for the workshop
+
+To participate in the workshop, please ensure that you have the following requirements:
+
+* Internet connection
+* Git
+* Go 1.19+
+* Make (for using Makefiles)
+* Text editor
+
+## Nice to have
+
+If you are using Visual Studio Code, we recommend installing the [Gno language extension](https://marketplace.visualstudio.com/items?itemName=harry-hov.gno) from the marketplace. This extension provides helpful features such as highlighting formatting errors and syntax highlighting for `.gno` files.
+
+## Workshop Commands
+### Initial Setup
+
+To clone the gno GitHub repository, run the following command:
+
+```bash
+git clone https://github.com/gnolang/gno.git
+```
+
+### Building/Installing the gno tools
+
+From the root of the cloned repository, navigate to gno.land and run the following commands:
+
+```bash
+cd gno.land
+make build && make install
+```
+
+Next, navigate to the `gnovm` directory and execute the following commands:
+
+```bash
+cd gnovm
+make build && make install
+```
```

Comment:
FYI, now we can do `go install github.com/gnolang/gno/gno.land/cmd/gnokey@latest`, just in case you see that easier for users.

Reply:
Nevermind, `gnoland` needs the genesis file that is located on the repo to work properly... Offtopic: shall we include the genesis file inside the binary when building?